src/main/java/impl/WriterToFileImpl.java

```diff
@@ -0,0 +1,17 @@
+package impl;
```

Comment:
This class should be placed in the `service.impl` package according to the project structure requirements.